channeldb/invoices.go

```diff
@@ -1311,6 +1319,14 @@ type InvoiceSlice struct {
 func (d *DB) QueryInvoices(q InvoiceQuery) (InvoiceSlice, error) {
 	var resp InvoiceSlice
 
+	// If MinCreationDate filter is greater than MaxCreationDate, we know
+	// there will be no matching results and can safely return early.
+	if q.MinCreationDate > 0 && q.MaxCreationDate > 0 {
+		if q.MinCreationDate > q.MaxCreationDate {
+			return resp, nil
```

Comment:
I'd error out here, specifying `Min > Max` is likley a bug in the calling code.

Reply:
Done.